osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/map.js

```diff
@@ -74,7 +84,7 @@
 
         this._setLocationFilterFromExcerptID = function(ID) {
             var that = this;
-            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/").on('data:loaded', function(){
+            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/?format=json").on('data:loaded', function(){
```

Comment:
No, it shouldn't. If it is indeed needed right now, I'd consider this a bug on the osamaxx rest api client. 

We probably just haven't set the settings:

```python
REST_FRAMEWORK = {
    'DEFAULT_RENDERER_CLASSES': (
        'rest_framework.renderers.JSONRenderer',
    ),
    'DEFAULT_PARSER_CLASSES': (
        'rest_framework.parsers.JSONParser',
    )
}
```

See: [REST Framework Settings](http://www.django-rest-framework.org/api-guide/settings/#settings)